integrated_channels/moodle/client.py

```diff
@@ -196,8 +184,8 @@ def _get_access_token(self):
                 'Content-Type': 'application/x-www-form-urlencoded',
             },
             data={
-                "username": decrypted_username if use_encrypted_user_data else username,
-                "password": decrypted_password if use_encrypted_user_data else password,
+                "username": self.enterprise_configuration.username,
```

Comment:
we are changing it now because `username` is now encrypted for every moddle config?